Xamarin.Forms.Maps.Android/MapRenderer.cs

```diff
@@ -10,311 +11,358 @@
 using Java.Lang;
 using Xamarin.Forms.Platform.Android;
 using Math = System.Math;
-using Android.Runtime;
-using System.Collections;
 
 namespace Xamarin.Forms.Maps.Android
 {
-	public class MapRenderer : ViewRenderer<Map,MapView>, 
-		GoogleMap.IOnCameraChangeListener 
+	public class MapRenderer : ViewRenderer<Map, MapView>,
+		GoogleMap.IOnCameraChangeListener
 	{
-		public MapRenderer ()
-		{
-			AutoPackage = false;
-		}
+		const string MoveMessageName = "MapMoveToRegion";
 
 		static Bundle s_bundle;
-		internal static Bundle Bundle { set { s_bundle = value; } }
+
+		bool _disposed;
+
+		bool _init = true;
```

Comment:
This PR is just to get the formatting in line with everything else; we're not worried about logic issues here.